src/net/mod.rs

```diff
@@ -1,5 +1,9 @@
-//! Net module holds all sockets
-
+mod connection;
+mod packet;
 mod udp;
 
+use self::connection::{Connection, ConnectionQuality };
```

Comment:
I would like to rustfmt this file if possible, I am using this version `rustfmt 0.99.1-stable (da17b689 2018-08-04)`